src/pages/campaigns/edit.tsx

```
@@ -0,0 +1,49 @@
+  const { id } = useParams();
+  const { user } = useAuth();
+
+  const [campaign, setCampaign] = useState<
+    CampaignWithRelations | null | string
+  >(null);
+
+  useEffect(() => {
+    const fetchCampaign = async () => {
+      try {
+        const data = await getSingleCampaignById(
+          parseInt(String(id), 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)